### PR TITLE
releng - increment versions and update dependencies

### PR DESCRIPTION
**Pull request by kapilt**
- releng - increment version for release
- releng - update dependencies
- releng - add git blame ignore for formatting commits

### CODE REVIEW

**Comment by kapilt** [comment]
looks like there has been some changes upstream to botocore re user agent handling to align to other sdk values
https://github.com/boto/botocore/pull/2977

File: .git-blame-ignore-revs
Comment (thisisshi) [inline .git-blame-ignore-revs:1]: TIL about .git-blame-ignore-revs

https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view

**Review by thisisshi — APPROVED** [review]
lgtm, thanks

File: .git-blame-ignore-revs
Comment (kapilt) [reply .git-blame-ignore-revs:1]: worth putting that link into the file for self discovery.

**Pull request merged**